packages/metadata-agent/types/index.d.ts

```
@@ -0,0 +1,293 @@
+  applyLabels(options: ApplyLabelsOptions): Promise<ApplyLabelsResult>;
+  removeLabels(options: ApplyLabelsOptions): Promise<ApplyLabelsResult>;
+  setProjectFields(options: any): Promise<any>;
+  getRateLimit(): Promise<RateLimit>;
+  handleRateLimit(): Promise<void>;
+  retry(fn: Function, options?: any): Promise<any>;
+}
+
+export interface APIClient {
+  createClient(options: AuthenticateOptions): GitHubAPIClient;
+  authenticateClient(options: AuthenticateOptions): Promise<GitHubAPIClient>;
+  GitHubAPIClient: typeof GitHubAPIClient;
```

ESLint

[error] 'GitHubAPIClient' is not defined.

(no-undef)